projects/plugins/crm/modules/woo-sync/admin/activation/welcome-to-woo-sync.php

```diff
@@ -29,6 +29,7 @@
 //  which now get merged into this one css file.
 wp_enqueue_style('jpcrm-woo-sync-welcome-wizard',         plugins_url('/css/jpcrm-woo-sync-welcome-wizard'.wp_scripts_get_suffix().'.css',JPCRM_WOO_SYNC_ROOT_FILE) );
 
+$style_handles = array( 'woozbswelcomebootstrap', 'jpcrm-woo-sync-welcome-wizard' );
 // dequeue admin bar css
 wp_dequeue_style('admin-bar-css'); 
```

Comment:
This dequeue isn't needed, but will remove the entire wizard in a future PR anyway.